### PR TITLE
ref(metrics): Add sentry.group.created with the platform tag.

### PR DESCRIPTION
**Pull request by JTCunning**
Am I doing this minimal import right? Kinda scared to get a new import into `event_manager`.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
> Am I doing this minimal import right? Kinda scared to get a new import into event_manager.

I mean, you'd typically just import `metrics`, then use `metrics.incr(...)`.

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -871,6 +872,8 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                     **kwargs
                 ), True
 
+            metrics_incr('group.created', skip_internal=True, tags={'platform': event.platform})
```
Comment (mattrobenolt) [inline src/sentry/event_manager.py]: So it's worth noting that platform may be `None`. Not sure if that'll be problematic if that's passed through, but may want to handle it as like, `event.platform or 'unknown'` ?

**Comment by JTCunning** [comment]
Yea so I didn't want to disrupt any patterns, and I'm just making sure that this is ok.

Is it worth importing the entire module?

**Review by mattrobenolt — APPROVED** [review]

**Comment by mattrobenolt** [comment]
> Is it worth importing the entire module?

Yeah, I think you should imo. If anything, because I use this to search for existing `metrics.incr` calls. Since it uses that pattern everywhere else.

File: src/sentry/event_manager.py [hunk 2]
```diff
@@ -871,6 +872,12 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                     **kwargs
                 ), True
 
+            metrics.incr(
```
Comment (dcramer) [inline src/sentry/event_manager.py]: i think we have an object.created metric fwiw

Comment (JTCunning) [reply src/sentry/event_manager.py]: We do, but it is not per-platform.

**Pull request merged**